src/Tester/SerializationTests.cs

```diff
@@ -143,6 +143,7 @@ public static void Serializer(object untypedInput, Orleans.Serialization.BinaryT
             public static object Deserializer(System.Type expected, global::Orleans.Serialization.BinaryTokenStreamReader stream)
             {
                 TestTypeA result = new TestTypeA();
+                DeserializationContext.Current.RecordObject(result);
```

Comment:
Generated serializers and built-in serializers now record the object before deserialization has completed, so we modify this test to reflect that. This test may be of limited value given @galvesribeiro's test which instead uses generated code.